tests/testthat/test-renderer1-tooltip.R

```diff
@@ -9,12 +9,12 @@ viz <-
   list(scatter=ggplot()+
        geom_point(aes(life.expectancy, fertility.rate,
                       colour=region, size=population,
-                      tooltip=paste(country, "population", population),
+                      tooltip=paste(country, "population", population), id = country,
                       key=country), # key aesthetic for animated transitions!
                   clickSelects="country",
                   showSelected="year",
                   data=WorldBank)+
-       geom_text(aes(life.expectancy, fertility.rate, label=country,
+       geom_text(aes(life.expectancy, fertility.rate, label=country, tooltip = country, id = country,
```

Comment:
this will produce invalid HTML.
id must be unique across all elements on the page.
please change, for example `aes(id=paste0("text_",country))`